src/test/lib/storage/index/partial_hash/partial_hash_index_test.cpp

```diff
@@ -68,90 +95,130 @@ TEST_F(PartialHashIndexTest, IndexCoverage) {
-  EXPECT_EQ(hash_map_accessor->second[0], (RowID{ChunkID{1}, ChunkOffset{3}}));
-
-  index_map->find(hash_map_accessor, "paper");
-  EXPECT_EQ(hash_map_accessor->second.size(), 1);
-  EXPECT_EQ(hash_map_accessor->second[0], (RowID{ChunkID{1}, ChunkOffset{5}}));
-
-  index_map->find(hash_map_accessor, "clock");
-  EXPECT_EQ(hash_map_accessor->second.size(), 1);
-  EXPECT_EQ(hash_map_accessor->second[0], (RowID{ChunkID{1}, ChunkOffset{6}}));
+  EXPECT_EQ(index_map->at("hotel").size(), 1);
+  EXPECT_EQ(index_map->at("hotel")[0], (RowID{ChunkID{0}, ChunkOffset{0}}));
+
+  EXPECT_EQ(index_map->at("delta").size(), 3);
+  EXPECT_EQ(index_map->at("delta")[0], (RowID{ChunkID{0}, ChunkOffset{1}}));
+  EXPECT_EQ(index_map->at("delta")[1], (RowID{ChunkID{0}, ChunkOffset{3}}));
+  EXPECT_EQ(index_map->at("delta")[2], (RowID{ChunkID{1}, ChunkOffset{1}}));
+
+  EXPECT_EQ(index_map->at("apple").size(), 1);
+  EXPECT_EQ(index_map->at("apple")[0], (RowID{ChunkID{0}, ChunkOffset{4}}));
+
+  EXPECT_EQ(index_map->at("charlie").size(), 2);
+  EXPECT_EQ(index_map->at("charlie")[0], (RowID{ChunkID{0}, ChunkOffset{5}}));
+  EXPECT_EQ(index_map->at("charlie")[1], (RowID{ChunkID{0}, ChunkOffset{6}}));
+
+  EXPECT_EQ(index_map->at("inbox").size(), 2);
+  EXPECT_EQ(index_map->at("inbox")[0], (RowID{ChunkID{0}, ChunkOffset{7}}));
+  EXPECT_EQ(index_map->at("inbox")[1], (RowID{ChunkID{1}, ChunkOffset{7}}));
+
+  EXPECT_EQ(index_map->at("hello").size(), 1);
+  EXPECT_EQ(index_map->at("hello")[0], (RowID{ChunkID{1}, ChunkOffset{0}}));
+
+  EXPECT_EQ(index_map->at("funny").size(), 1);
+  EXPECT_EQ(index_map->at("funny")[0], (RowID{ChunkID{1}, ChunkOffset{2}}));
+
+  EXPECT_EQ(index_map->at("names").size(), 1);
+  EXPECT_EQ(index_map->at("names")[0], (RowID{ChunkID{1}, ChunkOffset{3}}));
+
+  EXPECT_EQ(index_map->at("paper").size(), 1);
+  EXPECT_EQ(index_map->at("paper")[0], (RowID{ChunkID{1}, ChunkOffset{5}}));
+
+  EXPECT_EQ(index_map->at("clock").size(), 1);
+  EXPECT_EQ(index_map->at("clock")[0], (RowID{ChunkID{1}, ChunkOffset{6}}));
 }
 
 TEST_F(PartialHashIndexTest, Iterators) {
-  auto begin = index->cbegin();
-  auto end = index->cend();
+  auto begin = cbegin(index);
+  auto end = cend(index);
 
   auto begin_copy = begin;
   EXPECT_EQ(begin_copy, begin);
   ++begin_copy;
   EXPECT_NE(begin_copy, begin);
 
-  EXPECT_EQ(std::distance(begin, end), 14);  // Test size of index iterator.
-  EXPECT_EQ(std::distance(index->null_cbegin(), index->null_cend()), 2);  // Test size of NULL values index iterator.
-  EXPECT_NE(std::find(begin, end, RowID{ChunkID{0}, ChunkOffset{4}}), end);  // Test for not-existing value in iterator.
+  // Test size of index iterator.
+  EXPECT_EQ(std::distance(begin, end), 14);
+  // Test size of NULL values index iterator.
+  EXPECT_EQ(std::distance(null_cbegin(index), null_cend(index)), 2);
+  // Test for not-existing value in iterator.
+  EXPECT_NE(std::find(begin, end, RowID{ChunkID{0}, ChunkOffset{4}}), end);
```

Comment:
I'm not sure if I understand correctly what you are trying to do here. Based on line 168, you want to ensure that `RowID{ChunkID{0}, ChunkOffset{4}}` is in the iterator's range, correct? If this is correct, the comment does not match the code, does it?